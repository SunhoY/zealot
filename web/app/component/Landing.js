import React, {Component} from 'react';

const styles = {
    container: {
        width: "100%",
        height: "100%"
    },
    title: {
        width: "100%",
        paddingTop: "100px",
        textAlign: "center"
    },
    centerAlign: {
        width: "100%",
        display: "block",
        textAlign: "center",
    },
    baeminFont: {
        fontFamily: "baemin"
    },
    textLarge: {
        fontSize: "180px",
        color: "#ffffff"
    },
};

class Landing extends Component {
    render() {
        return (
            <div style={styles.container} className="background">
                <div style={styles.title}>
                    <div style={Object.assign(styles.baeminFont, styles.textLarge)}>아재력</div>
                    <div style={Object.assign(styles.baeminFont, styles.textLarge)}>테스트</div>
                </div>
            </div>
        );
    }
}

export default Landing;