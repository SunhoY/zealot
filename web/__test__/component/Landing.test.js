import React from 'react';
import Landing from '../../app/component/Landing';
import renderer from 'react-test-renderer';

describe('Landing Test', () => {
    let subject,
        subjectTree;

    beforeEach(() => {
        subject = renderer.create(
            <Landing />
        );

        subjectTree = subject.toJSON();
    });

    it("renders correctly", () => {
        expect(subjectTree).toMatchSnapshot();
    });
});
