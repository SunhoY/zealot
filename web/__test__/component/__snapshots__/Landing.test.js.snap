exports[`Landing Test renders correctly 1`] = `
<div
  className="background"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }>
  <div
    style={
      Object {
        "paddingTop": "100px",
        "textAlign": "center",
        "width": "100%",
      }
    }>
    <div
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "baemin",
          "fontSize": "180px",
        }
      }>
      아재력
    </div>
    <div
      style={
        Object {
          "color": "#ffffff",
          "fontFamily": "baemin",
          "fontSize": "180px",
        }
      }>
      테스트
    </div>
  </div>
</div>
`;
